viewer/src/Main.rs

```diff
@@ -281,10 +293,10 @@ impl App {
                             {
                                 draw_x = x;
                                 draw_y = y;
-                                draw_tag = zone.tag_data.tag;
-                                draw_len = zone.tag_data.stop - zone.tag_data.start;
+                                draw_data = zone.tag_data;
                             }
                         }
+                        println!("{0:?}", draw_data);
```

Comment:
looks like a leftover debug println

Reply:
oh, I guess since this is mouse down only and the idea is to keep it on stdout so you can cut & paste if need be this is fine